serving/docker/lmi.Dockerfile

```diff
@@ -58,7 +58,7 @@ ENV DJL_CACHE_DIR=/tmp/.djl.ai
 ENV LD_LIBRARY_PATH=/usr/local/lib/python${python_version}/dist-packages/nvidia/cudnn/lib/
 ENV PYTORCH_LIBRARY_PATH=/usr/local/lib/python${python_version}/dist-packages/torch/lib
 ENV PYTORCH_PRECXX11=true
-ENV PYTORCH_VERSION=${torch_version}
+ENV PYTORCH_VERSION=2.4.0
```

Comment:
Maybe use ${djl_torch_version}. Will this affect lmi's pytorch version? I assume lmi wheel installs the pytorch version so this env should not affect lmi-dist, right? @xyang16 @siddvenk

Reply:
This PYTORCH_VERSION is only used by JNI library.

Reply:
This is what I mentioned about pytorch JNI version and wheel version will diverge in the decouple meeting.